app/src/main/java/com/team41/boromi/auth/LoginFragment.java

```diff
@@ -0,0 +1,89 @@
+package com.team41.boromi.auth;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.team41.boromi.BookActivity;
+import com.team41.boromi.R;
+
+/**
+ * A simple {@link Fragment} subclass. Use the {@link LoginFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class LoginFragment extends Fragment {
+
+  // TODO: Rename parameter arguments, choose names that match
+  // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+  private static final String ARG_PARAM1 = "param1";
+  private static final String ARG_PARAM2 = "param2";
```

Comment:
What are these used for?

Reply:
These are the templates that android studio creates for you. I left it here as an example for whoever is working on this fragment on how to pass things into the fragment.